crates/gpui/src/platform/windows/platform.rs

```diff
@@ -279,7 +281,23 @@ impl Platform for WindowsPlatform {
 
     // todo!("windows")
     fn path_for_auxiliary_executable(&self, name: &str) -> Result<PathBuf> {
-        Err(anyhow!("not yet implemented"))
+        if name == "cli" {
```

Comment:
Why is this only being done for "cli"? Couldn't we just do this for everything?

Reply:
Sorry was mostly just a draft. Really just airing out my thoughts.  I wasn't sure how to implement this function besides the cli condition.

Reply:
Gotcha, don't worry too much about the review then :)